src/lib/types.ts

```diff
@@ -13,51 +16,64 @@ export type SupabaseClientOptions = {
   /**
    * The Postgres schema which your tables belong to. Must be on the list of exposed schemas in Supabase. Defaults to 'public'.
    */
-  schema?: string
-  /**
-   * Optional headers for initializing the client.
-   */
-  headers?: GenericObject
-  /**
-   * Automatically refreshes the token for logged in users.
-   */
-  autoRefreshToken?: boolean
-  /**
-   * Allows to enable/disable multi-tab/window events
-   */
-  multiTab?: boolean
+  api: {
```

Comment:
Make this `rest`? API is kinda broad (c.f. API key) and this only affects postgrest-js. cc @inian

Reply:
Sounds good, will do

Reply:
Thinking about this a bit more, what about `db` instead?